### PR TITLE
GitHub PlatformIO CI

### PR DESCRIPTION
**Pull request by andre-stefanov**

### CODE REVIEW

File: Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp [hunk 1]
```diff
@@ -123,8 +123,14 @@
 // HEADLESS CLIENT
 // If you do not have a LCD shield on your Arduino Uno/Mega, set this to 1 on the line below. This is
 // useful if you are always going to run the mount from a laptop anyway.
-#define HEADLESS_CLIENT 0
+#ifndef HEADLESS_CLIENT
+#define HEADLESS_CLIENT 1
```
Comment (ClutchplateDude) [inline Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp:127]: This should be zero still by default

File: Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp [hunk 2]
```diff
@@ -123,8 +123,14 @@
...
+#endif
 
+#ifdef __AVR_ATmega328P__
```
Comment (ClutchplateDude) [inline Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp:130]: Why are we forcing non-headless with a Mega? User might want to use the Mega in headless mode....

**Review by ClutchplateDude — CHANGES_REQUESTED** [review]

File: Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp (hunk 1 above)
Comment (andre-stefanov) [reply Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp:127]: damn, you are right ... this was just for testing -.- we need local configs here also

File: Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp (hunk 2 above)
Comment (andre-stefanov) [reply Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp:130]: this is the UNO architecture. mega would be ATmega2560

Comment (ClutchplateDude) [reply Software/Arduino code/OpenAstroTracker/Configuration_adv.hpp:130]: Ahh yes

**Review by ClutchplateDude — APPROVED** [review]

**Pull request merged**